app/Dockerfile

```diff
@@ -2,6 +2,6 @@ FROM python:3.10-slim
 WORKDIR /app
 ENV PATH "$PATH:/app"
 COPY requirements.txt requirements.txt
-RUN sed -i "s/http/https/g" /etc/apt/sources.list && apt-get update && apt-get install -y --no-install-recommends --fix-missing g++ gcc libxslt-dev libffi-dev make graphviz libgraphviz-dev && pip3 install -r requirements.txt
+RUN apt-get update && apt-get install -y --no-install-recommends --fix-missing g++ gcc libxslt-dev libffi-dev make graphviz libgraphviz-dev && pip3 install -r requirements.txt
```

Comment:
Don't think we need graphviz anymore or even install it. Let me know if you want me to remove it? 😄